posts/2024-07-26-crates-io-development-update.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "crates.io: development update"
+author: Tobias Bieniek
+team: the crates.io team <https://www.rust-lang.org/governance/teams/crates-io>
+---
+
+Since crates.io does not have releases in the classical sense, there are no release notes either. However, the crates.io team still wants to keep you all updated about the ongoing development of crates.io. This blog post is a summary of the most significant changes that have been made to crates.io in the past months.
+
+## `cargo install`
+
+When looking at crates like [ripgrep](https://crates.io/crates/ripgrep) you will notice that the installation instructions now say `cargo install ripgrep` instead of `cargo add ripgrep`. This change was made to make it easier for users to install crates that have binary targets. `cargo add` is still the correct command to use when adding a crate as a dependency to your project, but for binary-only crates like ripgrep, `cargo install` is the way to go.
```

Comment:
```suggestion
When looking at crates like [ripgrep](https://crates.io/crates/ripgrep) you will notice that the installation instructions now say `cargo install ripgrep` instead of `cargo add ripgrep`. The team made this change to make it easier for users to install crates that have binary targets. `cargo add` is still the correct command to use when adding a crate as a dependency to your project, but for binary-only crates like ripgrep, `cargo install` is the way to go.
```
I prefer active voice vs passive voice. Explanation [here](https://developers.google.com/tech-writing/one/active-voice).
You can even just us "we" instead of "the team".

You don't need to address this comment. I just wanted to express my preference. Maybe you can consider this style for the next blog posts :)

Reply:
I also noticed that sometimes you use "our" later in the document. So you use both active and passive voice, which is inconsistent.